web/VERSION

```diff
@@ -1 +1 @@
-4.2.dev25
```

Comment:
I was wondering what keeps updating the `web/VERSION` now as it should...

Didn't remember I recently added this `.git/hooks/pre-commit` to my local repository 😂:

```
#!/bin/sh
GITROOT=$(git rev-parse --show-toplevel)
python3 $GITROOT/web/set_version.py
git stage $GITROOT/web/VERSION
```